packages/react/src/primitives/Label/Label.tsx

```diff
@@ -13,7 +13,7 @@ const LabelPrimitive: Primitive<LabelProps, 'label'> = (
     <View
       as="label"
       className={classNames(ComponentClassNames.Label, className, {
-        'amplify-visually-hidden': visuallyHidden,
+        [ComponentClassNames.VisuallyHidden]: visuallyHidden,
```

Comment:
Double checked this works
<img width="456" alt="image" src="https://user-images.githubusercontent.com/11983489/157509143-781d885c-6cfa-4537-838d-a95ebef71125.png">

Reply:
Thank you for checking. So happy this works with classnames now (maybe I was just doing it wrong before, who knows!)